tests/unit/s2n_encrypted_extensions_test.c

```diff
@@ -14,15 +14,22 @@
  */
 
 #include "s2n_test.h"
+#include "testlib/s2n_testlib.h"
 
 #include "tls/s2n_tls.h"
 #include "tls/s2n_tls13.h"
+
+#include "tls/extensions/s2n_server_alpn.h"
+#include "tls/extensions/s2n_server_max_fragment_length.h"
+#include "tls/extensions/s2n_server_server_name.h"
 #include "tls/extensions/s2n_server_supported_versions.h"
 
 #include "error/s2n_errno.h"
 #include "stuffer/s2n_stuffer.h"
 #include "utils/s2n_safety.h"
 
+int s2n_encrypted_extensions_send_size(struct s2n_connection *conn);
```

Comment:
Should this be placed in the header file along with server_extensions_send_size?

Reply:
server_extensions_send_size() is not used externally but I'm fine with either